### DISCUSSION TITLE
Demos & Examples 🪐

### DISCUSSION THREAD

**Discussion by alvarosabu** [Tresjs/tres#211] [Ideas]
![Screenshot 2023-04-19 at 13 24 38](https://user-images.githubusercontent.com/4699008/233060491-be9060a7-b39d-4a34-913a-8b2920ec6798.png)

**Is your feature request related to a problem? Please describe.**
Now that v2 is more stable, is time to shine ✨😎. The value would be to provide users with quick easy examples so they can use them.

The idea would be to create more demos on the [Playground](https://github.com/Tresjs/playground) and link them into the docs, so we can have a `Examples` page with some fancy stuff like this https://docs.pmnd.rs/react-three-fiber/getting-started/examples

## Out of scope

This does not include Stackblitz demos

## Examples list

Examples show how to implement a certain feature, they are scoped and serve only one purpose, and the package name should be on the tags.

- [x] [Basic Demo](https://playground.tresjs.org/experiments/tres-basic)
- [ ] OrbitControls with a tweakpane to change different props
- [ ] Cameras (Perspective vs Orthographic)
- [x] [Materials](https://playground.tresjs.org/experiments/materials)
- [x] [Lights](https://playground.tresjs.org/experiments/lights)
- [x] [Animations] Basic example UseRenderloop (https://playground.tresjs.org/experiments/materials)
- [x] [Events](https://playground.tresjs.org/experiments/events)
- [x] [Text 3D](https://playground.tresjs.org/experiments/text-3d)
- [x] [GLTFModel](https://playground.tresjs.org/experiments/gltf-model)
- [ ] useGLTF
- [ ] FBXModel
- [ ] useFBX
- [ ] useAnimations a.k.a. AnimationMixer 
- [ ] Environment maps
- [x] [Basic Shaders](https://playground.tresjs.org/experiments/basic-shaders)
- [x] [Pam Camera Mouse](https://playground.tresjs.org/experiments/pam-camera-mouse)
- [x] [Manual post-processing](https://playground.tresjs.org/experiments/post-processing-manual)
- [ ] [Realistic Scenes](https://playground.tresjs.org/experiments/realistic)
 
## Demos (Showcase)

Demos are meant to show more complex scenes that consist of many different features and packages to show what TresJS is capable of

- [x] [Portal Journey](https://playground.tresjs.org/experiments/portal-journey)
- [x] [Low Poly Planet](https://playground.tresjs.org/experiments/lowpoly-planet)
- [x] [Glass/Plastic Material](https://playground.tresjs.org/experiments/glass-material)

**Comment by alvarosabu** [comment]
@JaimeTorrealba do you think you could add some of the amazing demos that you are creating on the playground, I would love to showcase them in the docs 😊

**Reply by JaimeTorrealba** [reply]
My man @alvarosabu You have a pending PR for the glass effect, please check it out 
I will add others

**Reply by alvarosabu** [reply]
On playground? OK, I will add it
